modules/howtos/pages/encrypting-using-sdk.adoc

```diff
@@ -5,24 +5,23 @@
 // :page-aliases: ROOT:encrypting-using-sdk.adoc
 
 [abstract]
-The Field Level Encryption library enables encryption and decryption of JSON fields.
-
-CAUTION: The Field-Level Encryption library for Couchbase Java SDK is currently a _pre-release_.
-It is not supported, but is available for development and experiment.
-It is planned that a supported, full release version will be available later this year.
-Refer to the https://github.com/couchbase/java-couchbase-encryption[GitHub repo] for the latest updates.
+A practical guide for getting started with Field-Level Encryption, showing how to encrypt and decrypt JSON fields using the Java SDK.
+For a high-level overview of this feature, see xref:concept-docs:encryption.adoc[].
 
 [#package]
 == Packaging
 
-The Couchbase Java SDK uses the https://github.com/couchbase/java-couchbase-encryption[java-couchbase-encryption^] library to provide support for encryption and decryption of JSON fields.
-It includes cryptographic algorithms and keyrings you can use out of the box, and provides a framework for implementing your own crypto components.
+The Couchbase Java SDK works together with the https://github.com/couchbase/java-couchbase-encryption[Java Couchbase Encryption^] library to provide support for encryption and decryption of JSON fields.
+This library includes cryptographic algorithms and keyrings you can use out of the box, and provides a framework for implementing your own crypto components.
 
-NOTE: This separation of the encryption library ensures that the SDK does not have a dependency upon an encryption library in general use --
-but it does mean you have to explicitly include this external dependency in your project configuration.
+NOTE: The encryption code is packaged as an optional library because it is released under a https://github.com/couchbase/java-couchbase-encryption/blob/master/LICENSE.md[different license^] than the Couchbase Java SDK.
+To use the encryption library, you have to explicitly include this dependency in your project configuration.
```

Comment:
nit: does this read better with a passive voice?
"To use the encryption library, this dependency must be included in the project configuration".
hmmm... not that I read it - not really.